src/general/OptionsMenu.tscn

```diff
@@ -1917,10 +1920,49 @@ margin_top = 667.0
 margin_right = 702.0
 margin_bottom = 671.0
 
-[node name="HBoxContainer6" type="VBoxContainer" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer"]
+[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer"]
 margin_top = 681.0
 margin_right = 702.0
-margin_bottom = 735.0
+margin_bottom = 706.0
+text = "MISC_FUN"
+
+[node name="HBoxContainer9" type="HBoxContainer" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer"]
+margin_top = 716.0
+margin_right = 702.0
+margin_bottom = 741.0
+
+[node name="Label" type="Label" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer9"]
+margin_right = 250.0
+margin_bottom = 25.0
+rect_min_size = Vector2( 250, 0 )
+text = "SCREEN_EFFECT"
+
+[node name="HSeparator" type="HSeparator" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer9"]
+margin_left = 254.0
+margin_right = 498.0
+margin_bottom = 25.0
+size_flags_horizontal = 3
+custom_styles/separator = SubResource( 9 )
+
+[node name="ScreenEffect" type="OptionButton" parent="CenterContainer/VBoxContainer/VBoxContainer/VBoxContainer/PanelContainer/Misc/ScrollContainer/MarginContainer/VBoxContainer/HBoxContainer9"]
```

Comment:
It looks like you haven't correctly updated the focus neighbours. JSONDebug still has unupdated `focus_neighbour_bottom` and there's probably other nodes around this control that need the focus to be updated. If they are not then controller and keyboard GUI navigation is broken.